### PR TITLE
Updated Setup.py to remove Pasteurize

### PR DESCRIPTION
**Pull request by vgangaprasad**
Remove the following logic as Python 2 is being retired globally starting in 2020

from setuptools.command.build_py import build_py
                          : 
                          : 
class PasteurizeBuildCommand(build_py):
    def run(self):
        call(["pip", "install", "future"])
        call(["pasteurize", "./pyramda"])
        build_py.run(self)
                          : 
                          : 
cmdclass={
        'build_py': PasteurizeBuildCommand
    },

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,8 +21,5 @@ def run(self):
     ],
     install_requires=['future'],
```
Comment (jackfirth) [inline setup.py]: Is `future` still needed to install things?

File: setup.py [hunk 2]
```diff
@@ -1,15 +1,6 @@
 from setuptools import setup
-from setuptools.command.build_py import build_py
 from subprocess import call
```
Comment (jackfirth) [inline setup.py]: I think this import can be removed now too

**Comment by jackfirth** [comment]
For the Travis CI issues, I think dropping 3.3 from the CI config is the right way to go. Could you also add the newer versions? I think we want every version on [this list](https://docs.travis-ci.com/user/languages/python/#specifying-python-versions) except for 2.7.

**Comment by jackfirth** [comment]
Also, thank you so much for this pull request! It's something that's been bothering me for literal years.

File: setup.py (hunk 1 above)
Comment (vgangaprasad) [reply setup.py]: Hi Jack, its not required, i will remove it.

Comment (vgangaprasad) [reply setup.py]: Hi Jack,

I have made the following changes: 
- In Travis.yml, removed and added the versions (in travis.yml) that are listed in the link that you provided,
- In Flatten.py, removed "from past.builtins import basestring" and added "from past.builtins import str" as basestring is not compatible with Python 3.
- In Setup.py, removed "from setuptools.command.build_py import build_py"

Thanks,
Vijay

**Comment by jackfirth** [comment]
CI is passing, so looks good to me 👍 Merging!

**Pull request merged**